services/src/handlers/mod.rs

```diff
@@ -78,7 +78,7 @@ pub fn get_token(req: &HttpRequest) -> Result<SessionId> {
     let scheme = Bearer::parse(header).map_err(|_| Error::Authorization {
         source: Box::new(Error::InvalidAuthorizationScheme),
     })?;
-    SessionId::from_str(scheme.token()).map_err(|err| Error::Authorization {
-        source: Box::new(Error::InvalidUuid { source: err }),
+    SessionId::from_str(scheme.token()).map_err(|_err| Error::Authorization {
+        source: Box::new(Error::InvalidUuid),
```

Comment:
Why don't we need the inner error anymore?
Could be good for debugging?

Reply:
I couldn't keep it, because the source error previously was from the datatypes (or operators) crate, but now it has to be in the services crate and that would mean I'd have to box it. I can do it, but basically this error is thrown if a string is no a uuid, so there is no real information to get here.

Reply:
Why? It wasn't part of the API because it was not part of the display string.

Reply:
the error returned from this function was a operators-crate error. Now, because of the changes in the IDs etc., it is a services-crate error. And that one I can no longer use as a source field in the error, because that would be self-referential, aka not allowed because you can no longer compute the size of the Error enum. As I said, I can box it and put it back but I don't see why we would need to know the UUID error, it should suffice to know that the input token is not a valid UUID